lib/circles/members/invite.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(req, res) {
+    // TODO assert that userId exists
+    // TODO assert that authenticated user can invite member
+    // TODO assert that userId is not already invited
+    // TODO assert that userId is not already a member
```

Comment:
Could you add another "TODO contact invited member and let them know they are invited" :-) ?